docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -324,33 +323,19 @@ Release
         git checkout -b release/flocker-${VERSION} origin/master
         git push origin --set-upstream release/flocker-${VERSION}
 
-   - Create a ``flocker-${VERSION}.rb`` file by copying the last recipe file and renaming it for this release.
-
-   - Update recipe file:
-
-     - Update the version number:
-
-       The version number is included in the class name with all dots and dashes removed.
-       e.g. ``class Flocker012 < Formula`` for Flocker-0.1.2
-
-     - Update the URL:
+   - Create a recipe file using a Flocker admin script:
```

Comment:
Add a note explaining how to install the flocker admin tools `pip install Flocker[release]`